api/config/cdr_config_staging.json

```diff
@@ -88,6 +96,20 @@
       "hasFitbitData": true,
       "hasCopeSurveyData": true,
       "accessTier": "registered"
+    },
+    {
+      "cdrVersionId": 7,
+      "isDefault": true,
+      "accessTier": "controlled",
+      "name": "CDR for Controlled Tier - DO NOT USE",
```

Comment:
WHy is this unsafe to use?

Reply:
The original intention with these names (from Test onwards) was to indicate that it was likely to be non-functional, in the unlikely case of stumbling across it accidentally.

We can probably remove this now :)

Reply:
I think I'll do that in a separate PR however

Reply:
updated